pkg/apic/apiserver/models/management/v1alpha1/APIService.go

```diff
@@ -110,7 +110,7 @@ func (res *APIService) AsInstance() (*apiv1.ResourceInstance, error) {
 	if err != nil {
```

Comment:
everything in this folder path is generated code, the code within should be added to the template and not directly here